$OpenBSD: patch-src_bootstrap_check_rs,v 1.1 2018/02/23 16:38:16 landry Exp $
pass RUSTC_ERROR_METADATA_DST as absolute path (8c9bf663d4a8074e4955d1ad3f85a3b82e946297).
Index: src/bootstrap/check.rs
--- src/bootstrap/check.rs.orig
+++ src/bootstrap/check.rs
@@ -980,7 +980,8 @@ impl Step for ErrorIndex {
         build.run(builder.tool_cmd(Tool::ErrorIndex)
                     .arg("markdown")
                     .arg(&output)
-                    .env("CFG_BUILD", &build.build));
+                    .env("CFG_BUILD", &build.build)
+                    .env("RUSTC_ERROR_METADATA_DST", build.extended_error_dir()));
 
         markdown_test(builder, compiler, &output);
     }
